app/assets/javascripts/mobile/mobile_comments.js

```diff
@@ -11,6 +11,8 @@
     initialize: function() {
       var self = this;
 
+      new Diaspora.MarkdownEditor(".comment_box");
```

Comment:
I followed the idioms of Backbone where the HTML element is passed to the constructor.